storage/src/main/java/org/apache/kafka/server/log/internals/IndexEntry.java

```diff
@@ -14,7 +14,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+package org.apache.kafka.server.log.internals;
 
-package kafka.log
-
-class CorruptIndexException(message: String) extends RuntimeException(message)
```

Comment:
It's weird that git thinks CorruptIndexException is rename to IndexEntry.

Reply:
Yes, I agree. Not sure we can do anything about this though.